src/catalog/catalog_accessor.cpp

```diff
@@ -88,6 +88,11 @@ std::vector<constraint_oid_t> CatalogAccessor::GetConstraints(table_oid_t table)
 
 std::vector<index_oid_t> CatalogAccessor::GetIndexes(table_oid_t table) const { return dbc_->GetIndexes(txn_, table); }
 
+std::vector<std::pair<common::ManagedPointer<storage::index::Index>, const IndexSchema &>>
+CatalogAccessor::GetIndexObjects(table_oid_t table) {
```

Comment:
If we're going to have both of these functions, it's probably more consistent for this to be `GetIndexes` and the one that returns just the OIDs be `GetIndexOids`.  This will keep it better aligned with `GetIndex` and `GetIndexOid` semantics.